website/Contribution/CONTRIBUTING.rst

```diff
@@ -1310,7 +1310,10 @@ changes. This part should be multiple lines no longer than 80 characters.
   's' to verbs: Fixes, Removes
 * Use tags as `listed in the Odoo Guidelines <https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html#tag-and-module-name>`_ with the following extensions:
 
-  - **[MIG]** for migrating a module
+  - **[MIG]** for migrating a module.
+  - **[OU-ADD]** for adding migration scripts. If the migration scripts are included directly with the migration/module improvement, there's no need to do a separate commit.
+  - **[OU-FIX]** for fixing migration scripts.
+  - **[OU-IMP]** for improving migration scripts.
```

Comment:
I wonder, what about if instead of adding new "tags", just use the existing ones together:
- `[MIG][ADD]`
- `[MIG][FIX]`
- `[MIG][IMP]`